dbaas/pxc/pxc.go

```diff
@@ -286,7 +261,7 @@ func (p *PXC) CheckStatus(data []byte, pass map[string][]byte) (dbaas.ClusterSta
 				return dbaas.ClusterStateError, []string{}, errors.Wrap(err, "marshal answer")
 			}
 			return dbaas.ClusterStateReady, []string{string(answer)}, nil
-		}
+		}*/
 		return dbaas.ClusterStateReady, []string{fmt.Sprintf(okmsg, st.Status.Host, pass["root"])}, nil
```

Comment:
Instead of `[]string` we can return any struct here. But it should be returned as some interface that implements `fmt.Stringer`. So logrus will use String()  method in case of TEXTForamtter and marshal struct in case of JSONFromatter.